src/win/tty.c

```diff
@@ -109,6 +115,8 @@ static char uv_tty_default_inverse = 0;
 
 void uv_console_init() {
   InitializeCriticalSection(&uv_tty_output_lock);
+  InitializeCriticalSection(&uv_read_console_trap_lock);
+  uv_read_console_done = CreateEvent(NULL, 1, 0, NULL);
```

Comment:
InitializeCriticalSection won't fail but CreateEvent could. I'll add a check and an abort() in case of failure.